apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.ts

```diff
@@ -1,9 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-key-info-demo',
   templateUrl: './key-info-demo.component.html',
 })
 export class KeyInfoDemoComponent {
-  public layout = 'vertical';
+  @Input()
+  public set layout(value: 'vertical' | 'horizontal') {
```

Comment:
Same comment about typing this with the new type.

Reply:
changed to type